gpg-interface.h

```diff
@@ -3,6 +3,7 @@
 
 #define GPG_VERIFY_VERBOSE	1
 #define GPG_VERIFY_RAW		2
+#define GPG_VERIFY_QUIET	4
 
```

Comment:
Let's add:
```
if (!(flags && GPG_VERIFY_QUIET))
   print_signature_buffer(&sigc, flags);
```
from dc543f174ea8a58778fb34a85ccb5dfefbac0581 to this commit, shall we?